.github/workflows/openhands-resolver.yml

```diff
@@ -185,25 +198,11 @@ jobs:
 
       - name: Install OpenHands
         uses: actions/github-script@v7
-        env:
```

Comment:
It seems moving the env variables to `echo` statements causes a regression that #5641 and #5897 addressed; could we move them back to the original env statements? 

Here is a sample [test logs](https://github.com/malhotra5/test-repo/actions/runs/12613178741/job/35150712378) that showcase the failure